src/asm/main.c

```diff
@@ -106,6 +106,7 @@ static struct option const longopts[] = {
 	{ "gfx-chars",        required_argument, NULL,     'g' },
 	{ "nop-after-halt",   no_argument,       NULL,     'H' },
 	{ "halt-without-nop", no_argument,       NULL,     'h' },
+	{ "includefile",      required_argument, NULL,     'I' },
```

Comment:
gcc has `-I`, `-iquote`, `-isystem`, or `-idirafter` for specifying include paths, and `-i` or `-include` for specifying pre-included files. We historically have `-i` or `--include` for specifying include paths, and don't use `-I` yet. I'd like to avoid changing the historical behavior just for the sake of making a new feature consistent with gcc, but it may be the best option.

Reply:
GCC does not support `-i`. `-I` should be aliased to it, if supported at all.

Reply:
You mean `--includefile` shouldn't have any corresponding single-letter argument? It would be the only option like that, and particularly verbose since the argument value is a whole filename. If you don't like `-I`, how about `-f`?